hedera-node/hedera-token-service/src/main/java/com/hedera/node/app/service/token/records/GenesisAccountRecordBuilder.java

```diff
@@ -43,4 +44,14 @@ public interface GenesisAccountRecordBuilder {
      */
     @NonNull
     GenesisAccountRecordBuilder memo(@NonNull final String memo);
+
+    /**
+     * Tracks the <b>net</b> hbar transfers that need to be applied to the associated accounts
+     * (accounts are specified in the {@code TransferList} input param)
+     *
+     * @param hbarTransfers the net list of adjustments to make to account balances
+     * @return this builder
+     */
+    @NonNull
+    CryptoTransferRecordBuilder transferList(@NonNull TransferList hbarTransfers);
```

Comment:
This feels wrong. We should probably just dispatch the `createAccount` right ?

Reply:
I thought the circumstances that made these synthetic accounts necessary was a situation where the accounts already _were_ created...?

Reply:
I guess we can't dispatch here. 🤔